tests/failpoints/mod.rs

```diff
@@ -9,10 +9,13 @@ extern crate slog_global;
 
 mod cases;
 
+use std::sync::Once;
+
+static INIT: Once = Once::new();
+
 fn setup<'a>() -> fail::FailScenario<'a> {
-    let guard = fail::FailScenario::setup();
-    test_util::setup_for_ci();
-    guard
+    INIT.call_once(test_util::setup_for_ci);
+    fail::FailScenario::setup()
```

Comment:
why fail scenarios are also set up multiple times?

Reply:
https://docs.rs/fail/0.3.0/fail/#usage-in-tests